server/game/core/Player.js

```diff
@@ -691,8 +691,9 @@ class Player extends GameObject {
         if (this.resetTimerAtEndOfRound) {
             this.noTimer = false;
         }
-
-        this.passedActionPhase = false;
+        if (!(this.game.initiativePlayer === this && this.game.isInitiativeClaimed)) {
+            this.passedActionPhase = false;
```

Comment:
`resetForActionPhase` is called at the beginning of each action phase. Don't we always need to set this to false for both players so that they can have actions that phase?

Reply:
I don't think we need to reset to false if the player claimed. That's why I added the check if the player claimed this phase it won't reset passedActionPhase to false once a new phase Starts the player will have it reset to false. It's basically to simulate the player who claimed to constantly pass.

Reply:
removed redundant check.